aiofreepybox/api/tv.py

```diff
@@ -0,0 +1,151 @@
+        '''
+        Create tv record generator
+        '''
+        return await self._access.post('pvr/generator/', tv_record_generator)
+
+    async def delete_finished_tv_record(self, record_id):
+        '''
+        Delete finished tv record
+        '''
+        await self._access.delete(f'pvr/finished/{record_id}')
+
+    async def delete_programmed_tv_record(self, record_id):
+        '''
+        Delete programmed tv record
+        '''
+        await self._access.delete(f'pvr/programmed/{record_id}')
+
+    async def delete_tv_record_generator(self, generator_id):
+        '''
+        Delete tv record generator
+        '''
+        await self._access.delete(f'pvr/generator/{generator_id}')
+
+    async def edit_finished_tv_record(self, record_id, finished):
+        '''
+        Edit finished tv record
+        '''
+        return await self._access.put(f'pvr/finished/{record_id}', finished)
+
+    async def edit_programmed_tv_record(self, record_id, tv_record):
+        '''
+        Edit programmed tv record
+        '''
+        return await self._access.put(f'pvr/programmed/{record_id}', tv_record)
+
+    async def edit_tv_record_generator(self, generator_id, tv_record_generator):
+        '''
+        Edit tv record generator
+        '''
+        return await self._access.put(f'pvr/generator/{generator_id}', tv_record_generator)
+
+    async def get_finished_tv_records(self):
+        '''
+        Get finished tv records
+        '''
+        return await self._access.get('pvr/finished/')
+
+    async def get_mycanal_token(self):
+        '''
+        Get mycanal token
+        '''
+        return await self._access.get('tv/mycanal_token')
+
+    async def get_programmed_tv_records(self):
+        '''
+        Get programmed tv records
+        '''
+        return await self._access.get('pvr/programmed/')
+
+    async def get_tv_bouquet(self):
+        '''
+        Get tv bouquet
+        '''
+        return await self._access.get('tv/bouquets/')
+
+    async def get_tv_bouquet_channels(self, bouquet_id='freeboxtv'):
+        '''
+        Get tv bouquet channels
+        '''
+        return await self._access.get(f'tv/bouquets/{bouquet_id}/channels/')
+
+    async def get_tv_channels(self):
+        '''
+        Get tv channels
+        '''
+        return await self._access.get('tv/channels/')
+
+    async def get_tv_default_bouquet_channels(self):
+        '''
+        Get tv default bouquet channels
+        '''
+        return await self.get_tv_bouquet_channels()
+
+    async def get_tv_program(self, program_id):
+        '''
+        Get tv program
+        '''
+        return await self._access.get(f'tv/epg/programs/{program_id}')
+
+    async def get_tv_program_highlights(self, channel_id, date=int(time.time())):
+        '''
+        Get tv program highlights
+        '''
+        return await self._access.get(f'tv/epg/highlights/{channel_id}/{date}')
```

Comment:
If `date` has the default value `time.time()`, it will be a floating point number representing the number of seconds since the epoch (for instance now it would be `1569875773.223918`), is that really what is supposed to be used in the url?

Reply:
yes, all dates are in the ``int``(timestamp) format

Reply:
Perfect, I missed the cast to `int`, all is good then 👍